### DISCUSSION TITLE
Settings not retained if the server change

### DISCUSSION THREAD

**Discussion by MrPisquan** [apohl79/audiogridder#237] [Help] [answered]
Hi!

First of all thank you for your incredible software. It is a real blessing for a lot of people.

The issue I am experiencing is related to the use of 2 servers. Both of them have the same plugins. The first is the same PC where I am working, the second one is a laptop connected via cable.

Sometimes the laptop has to work elsewhere, so I replace the server related to it with the server of my PC. When this happens, the plugin settings are not recalled correctly and the plugin is opened with its default preset.

I really hope there's a way to solve this, otherwise due my situation using a second computer is not possible.

Thank you again for you effort, it's really enormously appreciated

**Comment by CaBuffalo** [comment]
Good point. I haven't tried this.

**Comment by apohl79** [comment]
Are you able to reproduce this behaviour with a single plugin (or a small set..)? If so, could you follow the steps here: https://audiogridder.com/bug-reports/ ? And provide the log and trace files of the plugin and the server side?

Thanks!

**Comment by MrPisquan** [comment]
[FX.zip](https://github.com/apohl79/audiogridder/files/5630829/FX.zip)
[Server.zip](https://github.com/apohl79/audiogridder/files/5630831/Server.zip)
Hi,
this is a case where once the server change the plugin turns to grey. The plugin is installed in all the computers of the network.
It seems that this kind of problems affects mainly the VST 2.4 plugins, I have to do more testing.
I'm not sure which of the fx logs is the interesting one, I attach both of the FX logs recorded at a short distance. 
Thank you so much!
![Capture A](https://user-images.githubusercontent.com/75130976/100906313-f55fe600-34c0-11eb-8a93-05260317278d.PNG)
![Capture B](https://user-images.githubusercontent.com/75130976/100906317-f729a980-34c0-11eb-9b60-a6dcb9b1040c.PNG)

**Comment by apohl79** [comment]
Do you still have this problem in beta10?

**Reply by MrPisquan** [reply]
Yes, unfortunately the problem is still there...

**Reply by CaBuffalo** [reply]
Can you give this a try?

Have your session play while reconnect the server by in the server menu list?

**Comment by MrPisquan** [comment, marked as answer]
As far as I experienced, the problem is related to the Acustica Audio vst2 plugins. The vst3 ones work fine (apart the duplication of the servers - I have 4 computers in the network and for each one there are two servers displayed in the plugin, one working and the other not). I always change the server when the session is not playing to avoid any problem.
Thank you very much!

**Reply by MrPisquan** [reply]
Hi Andreas,
I'm proud to announce to you that the problem has disappeared with the excellent 11 release! Thank you so much, incredible work.

**Reply by CaBuffalo** [reply]
Awesome. This is great to hear.
